str/associatr/multiple_testing_correction/run_gene_level_pval.py

```diff
@@ -65,6 +64,7 @@ def cct(
     #' @export
     R code is implemented in python
     """
+    from scipy.stats import cauchy
```

Comment:
It'd be worth putting a comment on this so it doesn't get accidentally “tidied up” in future… something like:

```suggestion
    # Import here as a PythonJob's function must stand alone
    from scipy.stats import cauchy  # noqa: PLC0415
```

(The `noqa` will stop ruff from complaining about this if the relevant check gets applied by default in future.)